api/libretime_api/storage/views/third_party_track_references.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework import viewsets
+
+from ..models import ThirdPartyTrackReference
+from ..serializers import ThirdPartyTrackReferenceSerializer
+
+
+class ThirdPartyTrackReferenceViewSet(viewsets.ModelViewSet):
+    queryset = ThirdPartyTrackReference.objects.all()
+    serializer_class = ThirdPartyTrackReferenceSerializer
+    model_permission_name = "thirdpartytrackreference"
```

Comment:
This is the one model I could not really understand it's purpose and where it should be put.

From the DB schema, it has a tight relation between the celery tasks, the storage.File, and the podcast but what purpose it has I still wonder.

Am I right having this in storage and not in the Core app ?

Reply:
As far as I know, yes. But I don't really understand it either